tests/testthat/test-proxy.R

```diff
@@ -0,0 +1,22 @@
+test_that("eval_*() respects proxy settings", {
+  foo <- structure(list(), class = "foo")
+  local_bindings(
+    tidyselect_data_proxy.foo = function(x) {
+      data.frame(x = 1, y = 2)
+    },
+    tidyselect_data_supports_predicates.foo = function(x) {
+      FALSE
+    },
+    .env = globalenv()
```

Comment:
```suggestion
    }
```

Reply:
This is necessary, otherwise the S3 registration stuff doesn't work right.